alts/src/test/java/io/grpc/alts/ComputeEngineChannelBuilderTest.java

```diff
@@ -30,9 +27,5 @@
   public void buildsNettyChannel() throws Exception {
     ComputeEngineChannelBuilder builder = ComputeEngineChannelBuilder.forTarget("localhost:8080");
     builder.build();
-
```

Comment:
With this part removed, does this test serve any useful purpose? Is there a way we can verify the `protocolNegotiatorFactory` in the `delegate`?

Reply:
It does verify that the builder doesn't throw. I don't think the test really had much benefit before either.